cli/cfncluster/cfncluster.py

```diff
@@ -99,19 +103,26 @@ def create(args):
 
             if status != 'CREATE_COMPLETE':
                 logger.critical('\nCluster creation failed.  Failed events:')
-                events = cfnconn.describe_stack_events(stack)
+                events = cfn.describe_stack_events(StackName=stack_name).get('StackEvents')
                 for event in events:
-                    if (event.resource_status == 'CREATE_FAILED'):
+                    if event.get('ResourceStatus') == 'CREATE_FAILED':
                         logger.info("  - %s %s %s" %
-                                    (event.resource_type, event.logical_resource_id,
-                                     event.resource_status_reason))
+                                    (event.get('ResourceType'), event.get('LogicalResourceId'),
+                                     event.get('ResourceStatusReason')))
             logger.info('')
-            outputs = cfnconn.describe_stacks(stack)[0].outputs
+            outputs = cfn.describe_stacks(StackName=stack_name).get("Stacks")[0].get('Outputs')
             for output in outputs:
```

Comment:
if 'outputs' is none you will get an Exception
'NoneType' object is not iterable
check on outputs or use
for output in outputs or []:

Reply:
changed to include a default value in the get `.get('Outputs`, default = [])`